test/project_types/script/layers/application/create_script_test.rb

```diff
@@ -35,9 +39,16 @@
       .stubs(:for)
       .with(context, language, script_name)
       .returns(task_runner)
+
+    repo = ep.sdks.for(language).repo
+    type = ep.dasherize_type
+    domain = ep.domain
```

Comment:
I would ONLY use a class without stubbing it if that class has very trivial logic or a class that's well established such as like `String` or `Array` etc..  Such trivial class could be something that only has accessors and nothing else.  The best practice in unit testing to really limit the scope of what your testing to the 1 class in question.  One way to think about this is that if this unit test fails, it has to be because there is something wrong with this class.  As you can see here, what if `ep.dasherize_type` throws exception?